### ISSUE TITLE
Error in Firefox using gregorian calendar

### ISSUE THREAD

**Issue by mijamo** [js-temporal/temporal-polyfill#125]
Code sample:

```
Temporal.Now.plainDateTime('gregory', 'Europe/Stockholm').add({ days: 2 })
```

Error that happens `RangeError: Era a (ISO year 2022) was not matched by any era`

Tested in Firefox 96.0 on Ubuntu 20.04, with version `0.3.0`

I have seen it referenced here as well: https://github.com/novacbn/kahi-ui/issues/114#issuecomment-1014562921

This seems to be a follow up of #92 

I don't think this is a duplicate of the previous issues but my apologies if it is.

**Comment by justingrant** [comment]
Yep this is a dupe of https://github.com/tc39/proposal-temporal/issues/2020.

**Comment by justingrant** [comment]
Fixed in #127
